pom.xml

```diff
@@ -0,0 +1,184 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<project xmlns="http://maven.apache.org/POM/4.0.0" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0 http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+
+  <parent>
+    <groupId>com.hotels</groupId>
+    <artifactId>hotels-oss-parent</artifactId>
+    <version>2.0.5</version>
+  </parent>
+
+  <groupId>com.hotels</groupId>
+  <artifactId>housekeeping</artifactId>
+  <version>0.0.1-SNAPSHOT</version>
+
+  <properties>
+    <powermock.version>1.5.6</powermock.version>
+    <jasypt-spring-boot.version>1.5-java7</jasypt-spring-boot.version>
+    <hadoop.version>2.7.1</hadoop.version>
+  </properties>
+
+  <dependencies>
+    <!-- Hadoop -->
+    <dependency>
+      <groupId>org.apache.hadoop</groupId>
+      <artifactId>hadoop-common</artifactId>
+      <version>${hadoop.version}</version>
+      <scope>provided</scope>
+    </dependency>
+
+    <!-- DataSource (HikariCP) -->
+    <dependency>
+      <groupId>com.zaxxer</groupId>
+      <artifactId>HikariCP</artifactId>
+      <version>2.4.3</version>
+    </dependency>
+
+    <dependency>
+      <groupId>joda-time</groupId>
+      <artifactId>joda-time</artifactId>
+      <version>2.9.9</version>
```

Comment:
Version in master is 2.8.2, I'm fine with going for an upgrade as long as nothing breaks.